### PR TITLE
refactor(addon-commerce): use html whitespace sensitive for prettier

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [ ] Feature
-   [x] Refactoring
-   [ ] Code style update
-   [ ] Build or CI related changes
-   [ ] Documentation content changes

## What is the current behavior?

Need changes before upgrade to #988

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

**Comment by waterplea** [comment]
Need to fix that space :)

**Comment by splincode** [comment]
@waterplea review again, please)

File: projects/addon-commerce/components/money/money.style.less [hunk 1]
```diff
@@ -18,6 +18,10 @@
     }
 }
 
-.t-currency:not(:empty) {
-    margin-left: 0.25em;
+.t-currency {
+    display: inline-block;
```
Comment (waterplea) [inline projects/addon-commerce/components/money/money.style.less:22]: Try `inline-flex` on the `:host` instead

**Comment by splincode** [comment]
@waterplea updated

**Comment by waterplea** [comment]
Maybe let's just remove margin and keep the native space that appears now? I don't like to rely on that but we have screenshot testing which would show any issues if they appear.

**Comment by splincode** [comment]
@waterplea @vladimirpotekhin @nsbarsukov please, review me))

**Review by nsbarsukov — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**